todo_app/logs/app.log

```diff
@@ -0,0 +1,159 @@
+### 2022-02-27 18:52:11,808 ### appLog ### INFO ### Title to add: rweqrwer ###
```

Comment:
Don't commit log files. 

Add it to your .gitignore file and remove it from the repo (either delete it altogether and commit, or run `git rm --cached todo_app/logs/app.log`

Reply:
Added to .gitignore on top level and removed log.